app/main.py

```diff
@@ -1 +1,41 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
```

Comment:
ne attribute per line, when you have a lot of attributes. At checklist you have a good example of styling
```
    def long_function_name(
            var_one,
            var_two,
            var_three,
            var_four
    ) -> None:
```